test-data/unit/check-incremental.test

```diff
@@ -2285,7 +2289,7 @@ S = str
 import b
 def f(x: b.S) -> int: return 0
```

Comment:
Yes, they seemed to not add value when multiple test cases are very similar. I double checked and added those when they actually seemed useful.